src/libs/PolicyUtils.ts

```diff
@@ -520,10 +522,33 @@ function getDefaultApprover(policy: OnyxEntry<Policy>): string {
 /**
  * Returns the accountID to whom the given employeeAccountID submits reports to in the given Policy.
```

Comment:
This needs to be updated, as we're no longer passing `employeeAccountID`

Reply:
I updated.